gst/nnstreamer/tensor_source/tensor_src_iio.c

```diff
@@ -934,7 +935,7 @@ gst_tensor_src_iio_set_property (GObject * object, guint prop_id,
       break;
 
     case PROP_FREQUENCY:
-      self->sampling_frequency = g_value_get_uint64 (value);
+      self->sampling_frequency = (guint64) g_value_get_ulong (value);
```

Comment:
ulong and uint64 are mixed up here.

```sampling_frequency``` is changed from ulong to uint64
and you are changing some related uint64 back to ulong. Make it consistent.

Reply:
sampling_frequency is interfaced as ulong for outside and stored as uint64 internally.
value is received from the user, which should be long format, hence the use of g_value_get_ulong(). 
as we store the value in uint64, we type cast it to uint64.

All the locations where the code interacts outside the file, ulong type is maintained.
Code had many inconsistencies which are handled by this commit.

Reply:
Oh,. ok. then. LGTM.